src/cryptoadvance/specter/static/styles.css

```diff
@@ -702,29 +707,67 @@ tbody tr:hover{
 .tx{
     max-width: 200px;
 }
-.scroll{
+
+/* Scroll functionality along x axis
+   (mainly used for xpub keys)
+*/
+
+.scroll {
     padding: 0 10px;
 }
-.scroll > *{
+.scroll > * {
     /* ugly fix for firefox - scrolling child element */
     display: block;
     overflow-x: scroll;
     width: 100%;
     padding: 20px 0;
 }
-.scroll > *::-webkit-scrollbar,
-#side-content::-webkit-scrollbar,
-main::-webkit-scrollbar,
-pre::-webkit-scrollbar{
-  display: none;
-}
-/* Hide scrollbar for IE, Edge and Firefox */
-td.scroll > *,
-#side-content,
-main,pre{
+/* Hide scrollbar thumb and track for IE, Edge and Firefox */
+td.scroll > *, pre {
   -ms-overflow-style: none;  /* IE and Edge */
   scrollbar-width: none;  /* Firefox */
-  scrollbar-height: none;  /* Firefox */
+}
+/* Hide scrollbar thumb and track for Chrome */
+td.scroll > *::-webkit-scrollbar,
+pre::-webkit-scrollbar {
+    display: none;
+}
+
+/* Standard scrollbar along y axis
+scrollbar-color and scrollbar-width only supported by Firefox */
+
+main {
+  overflow-y: scroll;
+  scrollbar-color: hsl(40 100% 75%) hsl(40 100% 50%);
```

Comment:
The chosen colors are not appealing.  How about something like  
![image](https://user-images.githubusercontent.com/60378539/157189533-4db82a2e-b84a-4f83-8aa4-698fe963e75d.png)
it looks like
![image](https://user-images.githubusercontent.com/60378539/157189609-e767f185-4e3e-42e3-b91d-6a98026e5839.png)

Reply:
How can you not like orange :-). I wanted to have two distinct different colors for the two scrollbars. Let's do it like this I will discuss it with our design guy and present your suggestion.

Reply:
Addressed in https://github.com/cryptoadvance/specter-desktop/pull/1580/commits/05c5a62e0bac83eab31dfda5d2d935b3bb67adef